backend/endpoints/annotations/queries/getOverallStats.sql

```diff
@@ -0,0 +1 @@
+SELECT count(DISTINCT k.image_id) AS known_count, count(DISTINCT i.id) AS total_count FROM image i LEFT JOIN known k ON k.image_id = i.id;
```

Comment:
Annotated change:

```sql
SELECT
  count(DISTINCT a.image_id) AS annotated_count,
  count(DISTINCT i.id) AS total_count
FROM 
  image i
  LEFT JOIN image_annotation a ON a.image_id = i.id
```

Eventually I'll probably update this to count how many annotations are "complete" for each image